config/exchanges/AR_BR.yaml

```diff
@@ -0,0 +1,4 @@
+lonlat:
```

Comment:
Actually I don't think those are necessary.
The aggregation script will keep the exchanges that are need and set them to the parent level when the country level view is selected.

Reply:
Are you sure? We manually added the others (and I added the ones of the US).

Reply:
Actually yes you're right, I read too fast through the config files. So no changes need here :)